custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js

```diff
@@ -2279,6 +2279,16 @@ function AwcReportsController($scope, $http, $location, $routeParams, $log, DTOp
         vm.getDataForStep(vm.step);
     });
 
+    vm.isOlderThan90Days = function (date) {
```

Comment:
I believe putting this check on the backend where it sends this list of image data would have been a better option. here: https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/reports/awc_reports.py#L139.  Doing so will automatically have the FE with an empty list and then that can be easily checked and put message is list if empty.

@Cory thoughts?

Reply:
Oh, I didn't think of that. If it works that way that sounds good!

Reply:
@saiprakash-v  could you please address this comment. I think these kinds of validation better make sense to put on the backend wherever possible.

Reply:
Okay @Rohit25negi I will make the changes in backend